torch/autograd/_functions/tensor.py

```diff
@@ -21,6 +21,16 @@ def backward(ctx, grad_output):
                 return grad_output.type(ctx.input_type), None
 
 
+class Noop(Function):
```

Comment:
@colesbury How does Python autograd function work with our View logic? My guess is that this function is similar to an `input[:]` except that the output only shares storage but not has ViewImpl (a non-differentiable view if you use the terminology in #12502 ). I'm mainly worried that `rebase_history` of base will move this Function around so that the hooks registers on this is not the accurate grad_input/output. I'll need to think about this more.